ironic/drivers/modules/drac/raid.py

```diff
@@ -562,8 +599,8 @@ def _find_configuration(logical_disks, physical_disks):
                                         physical_disks_by_type,
                                         free_space_mb))
             if not result:
-                error_msg = _('failed to find matching physical disks for all '
```

Comment:
Upstream reviewers will probably ask you to revert this change since it's outside the scope of this patch.

Reply:
Reverted back to normal.